### ISSUE TITLE
ChemReg Tests on file upload

### ISSUE THREAD

**Issue by strets123** [thesgc/chembiohub_helpdesk#695]
Need to add tests on file upload. Will complete test of assayreg and also add test of chemreg file upload.

**Comment by KarenPorter** [comment]
Please expand.  What sorts of tests?

**Comment by strets123** [comment]
The test would ensure that the current functionality for uploading a file, mapping data and saving data then searching substructures works correctly and continues to work correctly after we have made the changes to harmonise data capture with assayreg and to support Scarab. 

The tests would be deployed as continuous integration with a number of files that have been used before.

New tests will be included in each sprint now until the end of the project such that we have some decent coverage of use cases. By limiting this to the file uploads we continue the work started previously and avoid too ambitious a goal.

**Comment by aekhendry** [comment]
Please discuss or mark as done as appropriate

**Comment by strets123** [comment]
Needs to be done as part of testing sprint

**Comment by strets123** [comment]
Now depends upon #908 to continue it.

**Comment by strets123** [comment]
moving back to backlog

**Comment by strets123** [comment]
Tests now pass but need to get CI working again

**Comment by strets123** [comment]
Test runner not updated

**Comment by PaulBarrett79** [comment]
For Andy's attention to estimate

**Comment by strets123** [comment]
This is not essential for deployment - stretch goal

**Comment by KarenPorter** [comment]
integration tests rework scheduled for today.

@strets123 this was estimated at 5 hours - now scheduled for a further 2 days. What explains the discrepancy?
